pkg/engine/operation/apply.go

```diff
@@ -89,7 +89,7 @@ func (ao *ApplyOperation) Apply(request *ApplyRequest) (rsp *ApplyResponse, st s
 	if status.IsErr(s) {
 		return nil, s
 	}
-	log.Infof("Apply Graph:%s", applyGraph.String())
+	log.Infof("Apply Graph:\n%s", applyGraph.String())
```

Comment:
log framework not add word wrap?

Reply:
The position of '\n' is before the graph string instead of the end of this log, it will make the whole graph easier to recognize.